docs/content/walkthrough.md

```diff
@@ -0,0 +1,139 @@
+---
+title: Walkthrough
+description: Understanding the Athens proxy and Go Modules
+menu: shortcuts
+---
+
+First, make sure you have [Go 1.11 installed](https://gophersource.com/setup/) and that GOPATH/bin is on your path.
+
+## Without the proxy
+Let's review what everything looks like in Go 1.11 without the proxy in the picture:
+
+```console
+$ git clone https://github.com/athens-artifacts/walkthrough.git
+$ cd walkthrough
+$ GO111MODULE=on go run .
+go: downloading github.com/athens-artifacts/samplelib v1.0.0
+The 🦁 says rawr!
+```
+
+The end result of running this command is that Go downloaded the package source and packaged
+it into a module, saving it in the Go Modules cache.
+
+Now that we know what stuff looks like without the proxy, only using Go Modules, let's take a look at
+how the proxy changes what we see.
+
+## With the proxy
+Using the most simple installation possible, let's walk through how to use the
+Athens proxy, and figure out what is happening at each step. 
+
+Before moving on, let's clear our Go Modules cache so that we can see the proxy
+in action without any caches populated:
+
+**Bash**
+```bash
+sudo rm -fr $(go env GOPATH)/pkg/mod
+```
+
+**PowerShell**
+```powershell
+rm -recurse $(go env GOPATH)\pkg\mod
+```
+
+Now use Go to install and run the Athens proxy in a background process:
+
+```console
+$ cd ..
+$ git clone https://github.com/gomods/athens.git
+$ cd athens
+$ go run ./cmd/proxy &
+[1] 25243
+INFO[0000] Starting application at 127.0.0.1:3000
+```
+
```

Comment:
Using `&` is a no-go in PowerShell :(

Should we use Start-Process -nonewwindow ?

I also think it needs to happen as an administrative shell? Here is my normal powershell output:

```
PS C:\Users\jerem\go\src\github.com\gomods\athens> go run .\cmd\proxy\
[36mINFO[0m[0000] Starting application at 127.0.0.1:3000
[31mERRO[0m[0117] exit status 1 : go: creating work dir: mkdir C:\WINDOWS\go-build251073827: Access is denied.
  [31mhttp-method[0m=GET [31mhttp-path[0m=/github.com/athens-artifacts/samplelib/@v/v1.0.0.info [31mhttp-url[0m=/github.com/athens-artifacts/samplelib/@v/v1.0.0.info [31mkind[0m="Internal Server Error" [31mmodule[0m=github.com/athens-artifacts/samplelib [31moperation[0m=download.versionInfoHandler [31mops[0m="[download.versionInfoHandler protocol.Info protocol.fillCache goget.Version goGetFetcher.Fetch module.getSources]" [31mversion[0m=v1.0.0
[36mhandler:[0m GET /github.com/athens-artifacts/samplelib/@v/v1.0.0.info [[91m500[0m]
```

Reply:
I think this is the issue:
```
tmp, err := ioutil.TempDir(os.Getenv("GOTMPDIR"), "go-build")
```


Probably need to set a non C:\WINDOWS directory for GOTMPDIR?

Reply:
Yes, c:\Windows will definitely require admin shell.

Reply:
Also Start-Process -NoWindow should work.

Reply:
agreed! start-process is the way to go, thanks for trying this out. 👍